src/modules/common/components/Selector.styles.js

```diff
@@ -14,12 +14,6 @@ export const Element = styled(Link)`
   border-radius: 30px;
   transition: background-color 0.2s, border-radius 0.2s;
   user-select: none;
-  
```

Comment:
You can remove the export and the `color` above, if you decide to specify the color for each item itself and export them separately